contracts/contracts/interfaces/IVault.sol

```diff
@@ -120,6 +120,10 @@ interface IVault {
 
     function priceUnitRedeem(address asset) external view returns (uint256);
 
+    function floorPrice() external view returns (uint256);
+
+    function floorPrice2() external view returns (uint256);
```

Comment:
Unused?

Reply:
I'm not sure where floorPrice2 came from. I've removed.